.github/workflows/ci.yml

```diff
@@ -43,6 +43,23 @@ jobs:
                 curl -L -O -k https://gitlab.tiker.net/inducer/ci-support/raw/main/build-and-test-py-project.sh
                 . ./build-and-test-py-project.sh
 
+    pytest_macos:
+        name: Pytest on Py${{ matrix.python-version }} (macOS)
+        runs-on: macos-latest
+        strategy:
+            matrix:
+                python-version: ['3.7', '3.x']
```

Comment:
This said that 3.7 is the oldest version
https://github.com/inducer/codepy/runs/6203139125?check_suite_focus=true

Reply:
Fine by me. We'll bump to 3.8 min in a month or two, I think.